### ISSUE TITLE
FuzzyProvider: Take count of matches into consideration when sorting

### ISSUE THREAD

**Issue by tinloaf** [atom-community/autocomplete-plus#345]
Hi,

I was wondering whether FuzzyProvider would be more useful if the number of matches (or fuzzy matches) would be taken into consideration for ordering the suggestions. As far as I can tell from the code, currently only the quality of the match decides the order of the suggestions. (Which means that there is no particular order at all for strict matching / perfect matches...)

I guess that when someone typed some word a lot of times, that's probably an indication that he meant that word again, even if the match is not as good as the match for another word, which he used only once?

**Comment by joefitzgerald** [comment]
Take a look at @benogle's PRs - he recently added similar behavior to SymbolProvider.

**Comment by benogle** [comment]
SymbolProvider has locality scoring, but not frequency scoring. The framework is there for adding new scoring mechanisms, though.

**Comment by joefitzgerald** [comment]
Considering FuzzyProvider has a short life ahead of it, I'm going to close this as wontfix.
